server/src/main/java/io/crate/statistics/ReservoirSampler.java

```diff
@@ -166,6 +167,11 @@ private Samples getSamples(List<Reference> columns,
                                Random random,
                                Metadata metadata) throws IOException {
 
+        if (LOGGER.isDebugEnabled()) {
+            LOGGER.debug("Start collecting samples for table: " + docTable.ident().fqn() + "(" +
+                columns.stream().map(r -> r.column().fqn()).collect(Collectors.joining(", ")) + ")");
```

Comment:
Could use parameter substitution, but no big deal.

And it looks like the other changes can be reverted

Reply:
yep, changed, thx.